src/soc/ibm/power9/homer.c

```diff
@@ -2121,8 +2145,39 @@ static void get_mem_cfg_msg_data(const struct occ_cfg_inputs *inputs,
 static void get_sys_cfg_msg_data(const struct occ_cfg_inputs *inputs,
 				 uint8_t *data, uint16_t *size)
 {
-	/* TODO: all sensor IDs are zero, because we don't have IPMI messaging,
-	 *       which seems to be required to get them */
+	/*
+	 * Array with sensors IDs filled with proper data for Talos II. Values come
```

Comment:
I'd rather avoid it until we can prove that these values are different, to avoid multiple copies of the same data in the tree, as it often happens currently. I can unhardcode this array and use `#define`d values for sensors IDs though, it should make it easier to spot the differences. Would that be OK?

Reply:
Still better than raw array

Reply:
Added.